pkg/mutation/mutator/mutator.go

```diff
@@ -0,0 +1,63 @@
+package mutator
+
+import (
+	"github.com/open-policy-agent/gatekeeper/pkg/mutation/path"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// ID represent the identifier of a mutation object.
+type ID struct {
+	Group     string
+	Kind      string
+	Namespace string
+	Name      string
+}
+
+// SchemaBindings represent the specific GVKs that a
+// mutation's implicit schema applies
+type SchemaBindings struct {
```

Comment:
Sorry, I got this wrong, it should be `SchemaBinding` singular, to match with the design of `applyTo`, which accepts a list of these objects:

https://docs.google.com/document/d/1_BlvsuBm3u0rKV57ch2_6yPfw50OLNreELdYOSGwcYI/edit#heading=h.rr4dwgisq9r7

Reply:
Will change.